examples/storm-kafka-examples/src/main/java/org/apache/storm/kafka/trident/TridentKafkaWordCount.java

```diff
@@ -95,7 +95,8 @@ public static void main(String[] args) throws Exception {
                 localSubmitter.submit(prodTpName, tpConf, KafkaProducerTopology.newTopology(zkBrokerUrl[1], topicName));
                 // Consumer
                 localSubmitter.submit(consTpName, tpConf, TridentKafkaConsumerTopology.newTopology(localSubmitter.getDrpc(),
-                        new TransactionalTridentKafkaSpout(newTridentKafkaConfig(zkBrokerUrl[0]))));
+                        new OpaqueTridentKafkaSpout(newTridentKafkaConfig(zkBrokerUrl[0]))));
+//                        new TransactionalTridentKafkaSpout(newTridentKafkaConfig(zkBrokerUrl[0]))));
```

Comment:
Example topology doesn't really need to provide cover features exhaustively. Users still can see TransactionalTridentKafkaSpout is available, even not looking at this class. If users can't find, IMHO it's a documentation issue.